README.md

```diff
@@ -2,4 +2,20 @@
 
 Helm is a tool that streamlines installation and management of Kubernetes applications. Think of Helm as an apt/yum/homebrew
 for Kubernetes. This repository provides learning materials for Helm.
- 
+
+The default branch of this repo is `master` and it uses the current major release of [Helm](https://helm.sh/) which is v3. If you
+are using Helm v2 then go to the [helm-v2](https://github.com/IBM/helm101/tree/helm-v2) branch. Please look below for more
+details on the different Helm releases.
```

Comment:
I don't think "Please look below for more
details on the different Helm releases." is needed OR suggest to re-write "Please refer to the Helem Status section below for more
details on the different Helm releases."

Reply:
Sounds good. Updated.